openapi/paths/application-instances@{applicationId}@configuration.yaml

```
@@ -0,0 +1,65 @@
+parameters:
+  - $ref: ../components/parameters/applicationId.yaml
+get:
+  x-products:
+    - Users
+  tags:
+    - Application users
+  summary: Retrieve an application instance configuration
+  operationId: GetApplicationInstanceConfiguration
+  x-sdk-operation-name: get
+  description: Retrieves installed application configuration.
```

suggestion:
```suggestion
  description: Retrieves installed application configuration with a specified ID.
```

---

```suggestion
  description: Retrieves configuration of an installed application with a specified ID.
```

ID of the application, not ID of the configuration. Is this correct?

---

Don't forget to add 'the' ('Retrieves the configuration')